woocommerce/woocommerce
# Add task list progress header

Partly addresses #32165 and #32163

Add progress header to task list, this is currently added as a customizable option given it will be used in multiple task lists.

### Screenshots

<img width="798" alt="Screen Shot 2022-03-09 at 4 09 59 PM" src="https://user-images.githubusercontent.com/2240960/157525623-0acc5555-a880-4332-baf3-99ff4455f3d0.png">

### Detailed test instructions:

- Enable the progress header by adding `'display_progress_header' => true` [here](https://github.com/woocommerce/woocommerce-admin/blob/main/src/Features/OnboardingTasks/TaskLists.php#L93).
- Start a new WooCommerce store and finish the onboarding flow
- Go to **WooCommerce > Home**
- The progress header should show up with a `Welcome to <site name>` 
- The store details will already be set as completed
- Visit a uncompleted task and go back to the home screen, the header should now show 'Let's get you started  🚀'
- Finish three tasks and go back to the home screen
- The header should now say 'You are on the right track' 
- The progress bar should also be updated and the piece of text `4 out of 7 complete` (numbers might be slightly different)
- Now finish 2 more tasks, the header should now say 'You are almost there'
- Now finish the task list, the progress header should disappear.
